spec/Failsafe.spec.ts

```diff
@@ -115,24 +118,24 @@ describe(`Failsafe`, function() {
         it(`executes scripts in a sequence`, async () => {
             const { run, logger } = failsafe();
```

Comment:
The previous test had multiple expect-include calls which actually did not check that the commands where run in the correct order. I've now used a single regex to match on the combined output… WDYT?